notebooks/vertex_genai/solutions/gemini_prompt_engineering.ipynb

```diff
@@ -0,0 +1,1031 @@
+{
```

Comment:
<p>This should come at the end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Thanks! Done! </p>